internal/envoy/v2/regex_test.go

```diff
@@ -29,19 +29,15 @@ func TestSafeRegexMatch(t *testing.T) {
 			regex: "",
 			want: &matcher.RegexMatcher{
 				EngineType: &matcher.RegexMatcher_GoogleRe2{
-					GoogleRe2: &matcher.RegexMatcher_GoogleRE2{
-						MaxProgramSize: protobuf.UInt32(maxRegexProgramSize),
-					},
+					GoogleRe2: &matcher.RegexMatcher_GoogleRE2{},
```

Comment:
This whole file could be deleted probably, these tests don't really test anything.